### ISSUE TITLE
Make it easier to modify charge quantities

### ISSUE THREAD

**Issue by reactormonk** [DarkFenX/Pyfa#314]
How I would do it: add an additional column with the number, and make that column editable.

**Comment by blitzmann** [comment]
I was actually just looking at this while working on #83. I do not think it is possible, or at least it is not straightforward, to do this. But I haven't taken more than a glance of it. I will look into it more thoroughly  when I've got time and report back. =)

**Comment by reactormonk** [comment]
I've only messed with tk, and there it's pretty simple. Let's see what wx is like.

**Comment by reactormonk** [comment]
Apparently it's possible, but you'll have to use a grid instead of a list. http://docs.wxwidgets.org/trunk/classwx_grid_cell_editor.html

**Comment by blitzmann** [comment]
Yeah, with the preliminary search last night it seems the plain old ListCtrl is not able - we'd have to switch to UltimateListCtrl or a Grid. But that's the thing. To switch over from ListCtrl... I'm not sure how involved that is. Note that all of out "lists" (fitting window, cargo, drones, market, projected items, etc) all utilize our base `Display` class which is based off of ListCtrl. To be an easy switch, they would have to have similar APIs (Ultimate may be better in this regard). Whether this is worth the effort I'm not sure.
 
I'll continue looking into this in my spare time. In the meantime, feel free to poke the source and if you want to undertake development on something like this, go for it. Even if you only get a rough proof of concept. If you have any questions about the source, let me know =)

**Comment by blitzmann** [comment]
Was going through the wxpython demos for another reason, and ran accross the edit mixin for list ctrl. 

http://www.wxpython.org/docs/api/wx.lib.mixins.listctrl.TextEditMixin-class.html

Apparently this allows inline editing of the ctrl. Dunno how practical it would be to apply this or if it would work out well, but it's there to look into when I've got time.
